ods_ci/utils/scripts/terraform/openstack/provision.py

```diff
@@ -3,7 +3,7 @@
 import shutil
 import sys
 
-from python_terraform import *
+from python_terraform import IsFlagged, IsNotFlagged, Terraform
```

Comment:
Do we actually still use this provision.py for anything? I wanted to test this in Jenkins and I can't find anything that runs this.

Reply:
we don't use it now, but we want to keep it